style.css

```diff
@@ -20,14 +21,97 @@ main {
   align-items: center;
 }
 
+.scrollable-place {
+  height: 1000px;
+}
+
 .nav-menu {
   display: none;
 }
 
+.open {
+  cursor: pointer;
+}
+
 .bars {
+  list-style: none;
+  color: #6070ff;
+  position: relative;
+}
+
+.bars button {
+  border: none;
+  color: #6070ff;
+  background: transparent;
+  display: block;
+}
+
+/* .bars button:active {
+  outline: none;
+  border: none;
+}
+
+.bars button:focus {
+  outline: 0;
+} */
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.-